controllers/welcome.rb

```diff
@@ -1,3 +1,89 @@
+# ===== welcome
 get '/' do
+  @apartments = Apartment.all
   erb :welcome
 end
+
+# ===== ===== ===== APARTMENTS ===== ===== =====
```

Comment:
I think we talked about this, but there's nothing technically wrong with putting all your controllers/routes in this one file, but convention is to split them up into logically separate chunks / files (i.e. one for tenants, one for apartments, etc).